brainstormforce/astra
# AST-474 - fix: hide on mobile/tablet for above footer is not working on frontend.

### Description
-  Responsive visibility function is missing in the above footer dynamic CSS that is why it is not working in the frontend only.
-  https://wp-astra.atlassian.net/browse/AST-474

### Screenshots


### Types of changes
Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Add components in the above footer builder and set the above footer hide on mobile or tablet.
- Check on frontend it is still visible.

### Checklist:
